java/clients/src/main/java/sleeper/clients/status/report/filestatus/StateStoreSnapshot.java

```diff
@@ -55,14 +55,14 @@ public List<FileInfo> getActive() {
         return active;
     }
 
-    public StateStoreReadyForGC getReadyForGC() {
-        return readyForGC;
+    public StateStoreFilesWithNoReferences getFilesWithNoReferences() {
+        return filesWithNoReferences;
     }
 
-    public static StateStoreSnapshot from(StateStore stateStore, int maxNumberOfReadyForGCFilesToCount) throws StateStoreException {
+    public static StateStoreSnapshot from(StateStore stateStore, int maxNumberOfFilesWithNoReferencesToCount) throws StateStoreException {
         return builder()
                 .active(stateStore.getActiveFiles())
-                .readyForGC(StateStoreReadyForGC.from(stateStore, maxNumberOfReadyForGCFilesToCount))
+                .filesWithNoReferences(StateStoreFilesWithNoReferences.from(stateStore, maxNumberOfFilesWithNoReferencesToCount))
```

Comment:
This is loading the active files twice. This will also result in a potentially inconsistent view, where if you call `StateStore.getAllFileReferences` you get a single (hopefully) consistent snapshot of the state.